infrastructure/core/locals.tf

```diff
@@ -13,5 +13,5 @@
 #  limitations under the License.
 
 locals {
-  subnet_address_spaces = cidrsubnets(var.core_address_space, 2, 2, 2, 2)
+  subnet_address_spaces = cidrsubnets(var.core_address_space, 3, 3, 2, 2, 2)
```

Comment:
Maybe a comment for clueless people like myself explaining why these numbers are set to what they are?